src/component/title.js

```diff
@@ -127,8 +128,9 @@ echarts.extendComponentView({
         var subTextEl = new graphic.Text({
             style: graphic.setTextStyle({}, subtextStyleModel, {
                 text: subText,
+                x: subtextStyleModel.get('x'),
```

Comment:
Are `x` and `y` attributes you add? Are they used for setting subtitle to be a different position to the title? In this case, I think `x` and `y` should be an absolute value, rather than the position relative to the title.
My suggestion is, when user has to set title and subtitle for totally different positions, they should use an array of titles for them. So there's no need to provide new attributes.